convert2rhel/toolopts.py

```diff
@@ -188,6 +189,11 @@ def _register_options(self):
             " to show you what rpm files have been affected by the conversion."
             % (PRE_RPM_VA_LOG_FILENAME, POST_RPM_VA_LOG_FILENAME),
         )
+        self._shared_options_parser.add_argument(
+            "--eus",
+            action="store_true",
+            help="Automatically recognize the system as eus, utilizing eus repos",
```

Comment:
Wondering if we should clarify in here that this doesn't affect 8.6 systems, only 8.8+

Reply:
Sure I can mention it